src/main/java/uk/gov/justice/digital/delius/service/ContactService.java

```diff
@@ -124,6 +123,24 @@ public void addContactForBulkCustodyKeyDateUpdate(final Offender offender, final
                 notesForKeyDatesUpdate(datesAmendedOrUpdated, datesRemoved));
     }
 
+    @Transactional
+    public void addContactForTierUpdate(Long offenderId, LocalDateTime date, String tier, String reason, Staff staff, Team team){
+        contactRepository.save(builder()
+            .contactDate(LocalDate.now())
+            .offenderId(offenderId)
+            .contactStartTime(LocalTime.now())
+            .notes(String.format(
+                "Tier Change Date: %s\n" +
+                "Tier: %s\n" + "Tier Change Reason: %s",DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss").format(date),tier,reason))
+            .staff(staff)
+            .staffEmployeeId(staff.getStaffId())
+            .teamProviderId(team.getTeamId())
+            .probationArea(team.getProbationArea())
+            .team(team)
+            .contactType(contactTypeRepository.findByCode("ETCH20").orElseThrow(() -> new NotFoundException("Cannot find contact type for tier update")))
```

Comment:
maybe extract ETCH20 as a constant so you can give it a readable name

Reply:
Done